static/css/marble.css

```diff
@@ -1096,17 +1084,46 @@ h6{
         display: inline-block;
         text-decoration: unset;
     }
-  
+
+    /*Node*/
+    .margin-node-menu-item {
```

Comment:
This is identical in all widths, move this out of the media widths.

Reply:
Moved this out of media queries.